sass/_base.sass

```diff
@@ -1121,6 +1121,16 @@ $feature-box-div-margin-bottom: 40px
     #vendorStrip
       display: none
 
+  # Add logo to CNCF section
+  section#cncf
+    padding-top: 60px
+    padding-bottom: 140px
+    background-image: url(/images/cncf-color.png)
+    background-position: center 100px
+    background-repeat: no-repeat
+    background-size: 300px
+
+
```

Comment:
hello @sftim.
This change works, though, looking at the other defines in the file, does it make sense to style the cncf section definition under a `section -> main` or `section` element? I am curious about the glossary PR related to the cncf id.
```
section
  #cncf
     ...
```

Reply:
This is my first encounter with SASS. I guessed at how it works, and I seem to have got roughly the right idea.

I _think_ the intent behind this rule is to put the CNCF logo on the home page without needing an `<img>` element. I'm trying to stick to that inferred intent.

I don't really know what `section #cncf` would mean in SASS. In plain CSS it wouldn't behave how I want.

Reply:
OK. Just tried again. I have not generated css files for some time,  did you generate the css files with this commit?
```
#cncf
  main
    padding-top: 60px
    padding-bottom: 140px
    background-image: url(/images/cncf-color.png)
    background-position: center 100px
    background-repeat: no-repeat
    background-size: 300px
```

Reply:
I've found out about generating CSS (I'd guessed that Netlify would do that).
and I've fixed an incorrect comment.

The change works _now_, I've tested it.